polaris-react/src/components/ProgressBar/ProgressBar.scss

```
@@ -59,7 +59,7 @@
 .colorCritical {
   // stylelint-disable -- Polaris component custom properties
   --pc-progress-bar-background: var(--p-color-bg-strong);
-  --pc-progress-bar-indicator: var(--p-color-bg-interactive-critical);
+  --pc-progress-bar-indicator: var(--p-color-bg-critical-strong);
```

```suggestion
  --pc-progress-bar-indicator: var(--p-color-bg-critical);
```

This value was originally `--p-interactive-critical` which [maps to `--p-color-bg-critical`](https://github.com/Shopify/polaris/blob/bb67c17da7838ccc460a50e1269848e236c8b29f/polaris-migrator/src/migrations/v11-styles-replace-custom-property-color/v11-styles-replace-custom-property-color.ts#LL174C4-L174C28)

---

Ah thanks @aaronccasanova 🙌🏽